README.md

```diff
@@ -2,28 +2,21 @@
 
 # Project Name
 
-> Description the project.
+> This is my first project with microverse,in this project i created a portifolio skeleton for mobile view,
 
+![screenshot](./images/skeleton.png)
+#
+# Built With
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+>HTML
+>css
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://misheck12.github.io/Portfolio/)
 
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
 ### Prerequisites
 
 ### Setup
```

Comment:
- [ ] [OPTIONAL] You can remove the headings without content and include them when content is available. This will make your README look more professional